official/resnet/keras/keras_imagenet_main.py

```diff
@@ -214,15 +207,12 @@ def run(flags_obj):
                       validation_freq=flags_obj.epochs_between_evals,
                       verbose=2)
 
-  if flags_obj.enable_e2e_xprof:
-    results = profiler.stop()
-    profiler.save(flags_obj.model_dir, results)
-
   eval_output = None
   if not flags_obj.skip_eval:
     eval_output = model.evaluate(eval_input_dataset,
                                  steps=num_eval_steps,
                                  verbose=2)
+  time_callback = callbacks[0]
```

Comment:
What do you think about a helper to get a specific call back by type as part of common instead of counting on the index?  Then the method can return value not found or something.  Assuming the first callback is time_callback feels risky.  

The rest looks like an improvement to me.

Reply:
Great suggestion! Fixed in the following commit.